cmd/traffic_cache_tool/CacheTool.cc

```diff
@@ -2346,9 +2349,11 @@ Check_Freelist(std::string devicePath)
         for (auto strp : sp->_stripes) {
           strp->loadMeta();
           strp->loadDir();
+          /* what is this trying to do?
```

Comment:
this was for checking for loops in freelist

Reply:
I'll leave it commented out until that's ready to go.